proxy/http/HttpConnectionCount.cc

```diff
@@ -21,51 +21,321 @@
   limitations under the License.
  */
 
+#include <algorithm>
+#include <records/P_RecDefs.h>
 #include "HttpConnectionCount.h"
+#include <ts/bwf_std_format.h>
 
-ConnectionCount ConnectionCount::_connectionCount;
-ConnectionCountQueue ConnectionCountQueue::_connectionCount;
+OutboundConnTrack::Imp OutboundConnTrack::_imp;
+// const std::chrono::seconds OutboundConnTrack::Group::ALERT_DELAY{60};
+
+OutboundConnTrack::GlobalConfig *OutboundConnTrack::_global_config{nullptr};
+
+const MgmtConverter OutboundConnTrack::MAX_CONV{
+  [](void *data) -> MgmtInt { return static_cast<MgmtInt>(*static_cast<decltype(TxnConfig::max) *>(data)); },
+  [](void *data, MgmtInt i) -> void { *static_cast<decltype(TxnConfig::max) *>(data) = static_cast<decltype(TxnConfig::max)>(i); },
+  nullptr,
+  nullptr,
+  nullptr,
+  nullptr};
+
+const MgmtConverter OutboundConnTrack::MATCH_CONV{
+  [](void *data) -> MgmtInt { return static_cast<MgmtInt>(*static_cast<decltype(TxnConfig::match) *>(data)); },
+  [](void *data, MgmtInt i) -> void {
+    //    *static_cast<decltype(TxnConfig::match) *>(data) = std::clamp(static_cast<decltype(TxnConfig::match)>(i), MATCH_IP,
+    //    MATCH_BOTH);
+    *static_cast<decltype(TxnConfig::match) *>(data) = static_cast<decltype(TxnConfig::match)>(i);
+  },
+  nullptr,
+  nullptr,
+  [](void *data) -> std::string_view {
+    auto t = *static_cast<OutboundConnTrack::MatchType *>(data);
+    return t < 0 || t > OutboundConnTrack::MATCH_BOTH ? "Invalid"sv : OutboundConnTrack::MATCH_TYPE_NAME[t];
+  },
+  [](void *data, std::string_view src) -> void {
+    OutboundConnTrack::MatchType t;
+    if (OutboundConnTrack::lookup_match_type(src, t)) {
+      *static_cast<OutboundConnTrack::MatchType *>(data) = t;
+    }
+  }};
+
+// Unfortunately these are not used in RecordsConfig.cc so that must be made consistent by hand.
```

Comment:
Not all I'd like, but at least the name is only in 2 places now, reducing the annoying typo glitches.